stdlib/stdlib.mli

```diff
@@ -1359,7 +1359,7 @@ module Oo           = Oo
 module Option       = Option
 module Parsing      = Parsing
 module Pervasives   = Pervasives
-[@@deprecated "Use Stdlib instead.\n\
+[@@deprecated "use the Stdlib module\n\
```

Comment:
this is the only use of `[@@deprecated` in the codebase (all other uses are `[@@ocaml.deprecated`) -- is this intentional?

Reply:
Fixed, thanks.